test/integration/job/job_test.go

```diff
@@ -58,6 +58,7 @@ import (
 	"k8s.io/kubernetes/test/integration/framework"
 	"k8s.io/kubernetes/test/integration/util"
 	"k8s.io/utils/pointer"
```

Comment:
I realize this is going to look at bit weird adding the `k8s.io/utils/ptr`.  I decided to open up #120951 for that reason.  Once that PR is merged, we can drop `k8s.io/utils/pointer`.  The linter was yelling at me because it doesn't want any new deprecated usage.